bot/exts/info/patreon.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+
+import arrow
+import discord
+from discord.ext import commands, tasks
+
+from bot import constants
+from bot.bot import Bot
+from bot.constants import Channels, Guild, Roles, STAFF_PARTNERS_COMMUNITY_ROLES
+from bot.decorators import in_whitelist
+from bot.log import get_logger
+from bot.utils.channel import get_or_fetch_channel
+from bot.utils.members import has_role_id
+
+log = get_logger(__name__)
+
+PATREON_INFORMATION = (
+    "Python Discord is not-for-profit and volunteer run, so we rely on Patreon donations to do what we do. "
```

Comment:
Minor, but "Python Discord is a volunteer led non-profit, so we rely..." sounds slightly better to me

Reply:
👍 I made a couple other small changes to it too